alohomora/src/rocket/json/json.rs

```diff
@@ -69,6 +76,18 @@ impl<T: RequestBBoxJson> RequestBBoxJson for HashMap<String, T> {
 // Anything that is json serializable can be made to be a response.
 macro_rules! impl_base_types {
     ($T: ty) => {
+        impl RequestBBoxJson for $T {
```

Comment:
Same as a above, we do not want to implement this for T directly, and the surrounding code has impls for it for BBox<T, ..>